src/iuse.cpp

```diff
@@ -8439,7 +8433,7 @@ int iuse::autoclave( player *p, item *it, bool t, const tripoint &pos )
 
         bool empty = true;
         item *clean_cbm = nullptr;
-        for( item &bio : it->contents ) {
+        for( item &bio : it->contents.all_items() ) {
             if( bio.is_bionic() ) {
                 clean_cbm = &bio;
```

Comment:
And this will lead directly to UB. `all_items` returns a `std::list<item>`, so the loop iterates over that list. But the list is never stored, so after the loop finishes, it's deleted (including all the contained items), and `clean_cbm` will now point to a deleted object.